### PR TITLE
Remove parts-meta-info and unify return from get_version

### PR DESCRIPTION
**Pull request by Hook25**
## Description

This unifies the `--version` output between snaps, native and debs. 

## Resolved issues

Resolves: https://warthogs.atlassian.net/browse/CHECKBOX-618

## Documentation

N/A

## Tests

Tested with the `checkbox-core-snap` built on this branch and `checkbox-uc22`. Version output is:
```
series22>  checkbox.checkbox-cli --version
checkbox-ng: 2.9.dev33+gb3023a28d
checkbox-support: 2.9.dev33+gb3023a28d
```

### CODE REVIEW

File: checkbox-ng/checkbox_ng/version.py [hunk 1]
```diff
@@ -31,63 +31,30 @@
...
+    info["checkbox-support"] = support_ver
     ignored_providers = [
-        'com.canonical.plainbox:manifest',
-        'com.canonical.plainbox:exporters',
-        'com.canonical.plainbox:categories',
+        "com.canonical.plainbox:manifest",
+        "com.canonical.plainbox:exporters",
+        "com.canonical.plainbox:categories",
     ]
     # maybe filter-out the built-in providers (exporters, categories, etc.)
```
Comment (yphus) [inline checkbox-ng/checkbox_ng/version.py:56]: I'd even remove this for loop for providers, they all come from the monorepo.

File: checkbox-ng/checkbox_ng/version.py [hunk 2]
```diff
@@ -31,63 +31,30 @@
...
-def get_stack_version():
     """
     Generate version information of Checkbox stack components as understood
     by Checkbox.
     """
     info = OrderedDict()
-    info['checkbox-ng'] = checkbox_ng.__version__
+    info["checkbox-ng"] = checkbox_ng.__version__
     try:
         import checkbox_support
```
Comment (yphus) [inline checkbox-ng/checkbox_ng/version.py:41]: Let's just report the version of checkbox-ng here.

**Review by yphus — CHANGES_REQUESTED** [review]
More cleanup suggestions, see inline

File: checkbox-ng/checkbox_ng/version.py (hunk 2 above)
Comment (Hook25) [reply checkbox-ng/checkbox_ng/version.py:41]: Agree

**Review by yphus — APPROVED** [review]
LGTM

**Pull request merged**